lib-rca/src/main/java/com/facebook/openwifi/librca/stats/inputs/InputStats.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.stats.inputs;
+
+/** Input Data Model */
+public class InputStats {
+    /** Radio parameters */
+    public static class Radio {
+        public long active_ms;
+        public long busy_ms;
```

Comment:
Can you change all the snake case in all these files to camel case? It does not have to be the same as field names in OpenWiFi